src/components/radio/radio_button.ts

```diff
@@ -0,0 +1,269 @@
+import {
+  Attribute,
+  Component,
+  ElementRef,
+  Host,
+  HostBinding,
+  HostListener,
+  Input,
+  OnInit,
+  Optional,
+  Output,
+  SkipSelf,
+  ViewEncapsulation,
+  ViewChild
+} from 'angular2/core';
+
+import {isPresent, StringWrapper, NumberWrapper} from 'angular2/src/facade/lang';
+import {ObservableWrapper, EventEmitter} from 'angular2/src/facade/async';
+import {Event, KeyboardEvent} from 'angular2/src/facade/browser';
+
+import {MdRadioDispatcher} from './radio_dispatcher';
+
+// TODO(mtlin):
+// Ink ripple is currently placeholder.
+// Determine motion spec for button transitions.
+// Design review.
+// RTL
+
+var _uniqueIdCounter: number = 0;
+
+@Component({
+  selector: 'md-radio-group',
+  host: {
+    'role': 'radiogroup',
+  },
+  templateUrl: './components/radio/radio_group.html',
+  styleUrls: ['./components/radio/radio-group.css'],
+  encapsulation: ViewEncapsulation.None
+})
+export class MdRadioGroup implements OnInit {
+  /** The value for the radio group. Should match currently selected button. */
+  value_: any;
+
+  /** The HTML name attribute applied to radio buttons in this group. */
+  @Input() name: string;
+
+  /** Child radio buttons. */
+  radios_: MdRadioButton[];
+
+  /** Disables all individual radio buttons assigned to this group. */
+  disabled_: boolean;
+
+  /** The currently selected radio button. Should match value. */
+  selected_: MdRadioButton;
+
+  /** Event emitted when the group value changes. */
+  @Output() change: EventEmitter<any>;
```

Comment:
For now, how about we use `string` since that's the basic value for a native input?